clients/merge-nethermind/mapper.jq

```diff
@@ -95,6 +95,7 @@ def clique_engine:
     "eip155Transition": env.HIVE_FORK_SPURIOUS|to_hex,
     "maxCodeSizeTransition": env.HIVE_FORK_SPURIOUS|to_hex,
     "maxCodeSize": 24576,
+    "maximumExtraDataSize": env.HIVE_MAX_EXTRA_DATA_SIZE|to_hex,
```

Comment:
Can we just hard-code this please? Adding new environment parameters for client-specific tweaks is discouraged.

Reply:
Changed to a hard-coded value, let me know if looks ok.